controllers/resources.go

```diff
@@ -1072,6 +1061,49 @@ func (r *DurosReconciler) deployCSI(ctx context.Context, projectID string, scs [
 			return err
 		}
 
+		if sc.Encryption && r.shootK8sVersionGreater120() {
```

Comment:
this is wrong, make i again as it was above and configure the CRD with a SC which has encryption set to enabled.